knowledge/style-guide.md

```diff
@@ -15,94 +15,136 @@ parent: Knowledge
 {:toc}
 </details>
 
-This document will present the officially supported Bedrock-Wiki style-guide for addon-creation. The goal of this guide is to promote best-practices while creating addons, and create a consistent format for everyone to follow.
+This document will present the officially supported Bedrock-Wiki style-guide for addon-creation. The goal of this guide is to promote best-practices
+while creating addons and create a consistent format for everyone to follow.
```

Comment:
Did a newline get added here?

Reply:
It move part of the line to line 19, my formatter went off :#